core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java

```diff
@@ -405,6 +405,10 @@ private boolean parseUrl(
             skip = 0;
         }
 
+        if (configuration.getMaxQueryResponseRowLimit() != Long.MAX_VALUE && (stop - skip) > configuration.getMaxQueryResponseRowLimit()) {
```

Comment:
same, comparison to MAX_VALUE is redundant

Reply:
Thats true, but most of the time maxQueryResponseRowLimit will be  Long.MAX_VALUE, I wanted to avoid the additional subtraction